docs/teststand.md

```
@@ -0,0 +1,152 @@
+
+The **Python install manager** (`py` command, pre-installed on TAMU lab machines as a Windows app alias) can install a real Python to your user profile without admin rights.
+
+Open a terminal and run:
+
+```
+py install 3.12
+```
+
+This installs Python 3.12 to:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)